app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,12 @@
+
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @user,url: sessions_path, html: {class: "form-horizontal"} do |f|
```

Comment:
I just noticed this is for `sessions/new.html.haml` - here you should use `form_tag`, because there is no session model and this is not model-backed form.